pkg/doc/jose/decrypter.go

```diff
@@ -93,6 +94,16 @@ func (jd *JWEDecrypt) Decrypt(jwe *JSONWebEncryption) ([]byte, error) {
 		return nil, fmt.Errorf("jwedecrypt: %w", err)
 	}
 
+	if len(recWK) == 1 {
+		// ensure EPK is marshalled the same way as during encryption since it is merged into ProtectHeaders.
```

Comment:
https://tools.ietf.org/html/rfc7518#section-4.6.1

> The "epk" (ephemeral public key) value created by the originator for
>    the use in key agreement algorithms.  This key is represented as a
>    JSON Web Key [JWK] public key value.  It MUST contain only public key
>    parameters and SHOULD contain only the minimum JWK parameters
>    necessary to represent the key

I believe the implication is that use should not be in the epk header since it is not part of the minimum parameters to represent the key.

Reply:
removed `use` JWK header in `epk`